### ISSUE TITLE
Correct return type for method get()

### ISSUE THREAD

**Issue by mrcnpdlk** [skipperbent/pecee-pixie#3]
QueryBuilderHandler.php
I think that get() method should return **\stdClass|array|null** or simply **mixed**.
```
/**
     * Get all rows
     * @throws Exception
     * @return \stdClass|null
     */
    public function get()
    {
```

**Comment by skipperbent** [comment]
You're right - however according to http://php.net/manual/en/pdostatement.fetchall.php the `fetchAll` returns `array` so i guess a more appropriate return type would be `array|null`.

I'll fix this in the next upcoming release - thanks for reporting the issue.

**Comment by mrcnpdlk** [comment]
Dont forget, that get() also can return "$eventResult" - this, according to documentation can by mixed type.
```
$eventResult = $this->fireEvents('before-select');
        if ($eventResult !== null) {
            return $eventResult;
        }
```

**Comment by skipperbent** [comment]
This issue has been resolved in version 3.1.0.

I really appreciate your feedback!

**Comment by mrcnpdlk** [comment]
Great work! I have just upgraded library and it seems to be OK!.
